### DISCUSSION TITLE
Could not create the task       Open the Browser Console to get details

### DISCUSSION THREAD

**Discussion by celstnblacc** [cvat-ai/cvat#2947] [Q&A]
Hello,
I am in debug mode with vscode (server: debug) (**http://localhost:3000/tasks/create**?projectId=4)
I created a project, now it ry to create a task
I always get the popup error message

Could not create the task
Open the Browser Console to get details

Can you tell me why ?
Thank you
<img width="1176" alt="Screen Shot 2021-03-15 at 2 06 02 AM" src="https://user-images.githubusercontent.com/3260645/111110491-03e38200-8533-11eb-9a8e-b105a4fc74fc.png">

**Comment by azhavoro** [comment]
Hi, the task cannot be created for a reason, what is the error in the browser console?
